test/functional/fixtures/es_archiver/dashboard/current/kibana/mappings.json

```diff
@@ -1,13 +1,74 @@
 {
   "type": "index",
   "value": {
-    "index": ".kibana",
+    "aliases": {
```

Comment:
Not sure about these changes:
* Just seems like it isn't right that adding new dashboard to dataset causes so much changes.  🤔 
* Also in `data.json.gz` there 8.0.0 version mentions now and I guess it could cause issues with backport 

I assume that something is not right with used method: (https://github.com/elastic/kibana/pull/66979/files#diff-72924cf33ef20e70416fdfb75f2a6afb)

@elastic/kibana-qa could you help 🙏 ? I wonder what is the best way to add a new dashboard to existing data? Or are these changes OK?

Reply:
I followed @spalger suggestion on regenerating the mappings
I'm definitely open to alternative ways of doing this :)

Reply:
I tried to do something very similar in this pr and with minimal changes: https://github.com/elastic/kibana/pull/67280
To not touch the mappings and have minimal changes to data, I just add new dashboard in 6.3.0 format (following example of other dashboard). 

Not sure if this is the best approach, but looks like it works

Reply:
I don't want to add it manually in the json, but to use the existing tooling.
Lets see what spencer says.

Reply:
So, it looks like this was generated with coach enabled rather than using the `node scripts/functional_tests_server` for OSS. That said, this archive is pretty old so I'm not surprised to see some new properties on the archive. The alias stuff shouldn't be a problem since we trigger a migration after importing a .kibana index anyway, but the extra properties in the mappings by could be removed and I wouldn't complain. You might want to consider starting an OSS server, loading the archive into that server, and then archiving it again after it is migrated for OSS?

Reply:
@spalger this is exactly how I generated this one.
Started an OSS tests server, loaded the data and saved it.
This is the resulting output.

Reply:
Sounds good, I don't expect it to cause any issues then